### PR TITLE
[Feature] Organization's Locations Management

### PR DESCRIPTION
**Pull request by vitokhangnguyen**
This PR includes:

* Adding CRUD of Locations for an Organization
* Adding Google Map API to view Locations
* Fixing bugs in inventory adjustment feature

### CODE REVIEW

File: src/app/components/views/location-management-page/location-list-pane/location-list-pane.component.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                <app-location-list-item
+                    *ngFor="let locItem of locations"
+                    [location]="locItem"
+                    (click)="onSelectLocation(locItem)"
+                    (editButtonClick)="onLocationEditInitiate($event)"
+                    (deleteButtonClick)="onLocationDelete($event)"
+                ></app-location-list-item>
+            </div>
+        </ng-template>
+        <ng-template #emptyList>
```
Comment (Silvyre) [inline src/app/components/views/location-management-page/location-list-pane/location-list-pane.component.html]: I'm not able to test this fully for the reasons stated in https://github.com/vitokhangnguyen/InvenTurbo-backend/pull/11#discussion_r396013891.

However, I can say that this `<ng-template>` tag is _not_ rendering in [Ikea](http://localhost:8080/api/organization/Ejp7mlGIx6cQ1j9ZNIWb)'s case; the spinners are just spinning forever.

I'll tentatively approve, assuming this works better for other organizations.

Comment (vitokhangnguyen) [reply src/app/components/views/location-management-page/location-list-pane/location-list-pane.component.html]: I fixed the problem at https://github.com/vitokhangnguyen/InvenTurbo-backend/pull/11#discussion_r396013891 so this should work now.

On a side note, please be aware that my PR requires an extra dependency (Google Map API) so remember to do `npm i` before you test.

Test again when you have time!

Comment (Silvyre) [reply src/app/components/views/location-management-page/location-list-pane/location-list-pane.component.html]: Resolved, thanks!

**Comment by vitokhangnguyen** [comment]
@Silvyre Thank you for testing out, again. You can play around with the Create/Edit and Delete features too if you want. I will land this PR maybe on Monday.

**Pull request merged**